docs/docs/guides/installation.mdx

```diff
@@ -20,19 +20,31 @@ Please ensure the systems on which you want to install Infrahub meet the [hardwa
 
 ## Quick start via curl
 
-To quickly spin up Infrahub locally:
+To quickly spin up the latest Infrahub locally:
 
 ```shell
 curl https://infrahub.opsmill.io | sudo docker compose -f - up -d
 ```
 
-Alternative examples:
+or
+
+```shell
+curl https://infrahub.opsmill.io/stable | sudo docker compose -f - up -d
+```
+
+Alternatively, you could indicated `develop` for or any version in the URL:
 
 ```shell
 curl https://infrahub.opsmill.io/develop | sudo docker compose -f - up -d
-curl https://infrahub.opsmill.io/0.15.2 | sudo docker compose -f - up -d
+curl https://infrahub.opsmill.io/1.0.7 | sudo docker compose -f - up -d
 ```
 
+:::warning
+
+We do not recommend using the development version (`develop` branch) for production purposes or as an introduction to Infrahub.
```

Comment:
Should we add a link to our releases page notating to check and make sure the version is one of the newer ones? I'm afraid of having specific versions here as we'd have to keep this up to date or add a note.

I guess we could add this as part of the version bump process as well?

Reply:
Non-blocking comment, but figured I'd at least mention my thinking :)

Reply:
Good idea for the link, gonna add it